### PR TITLE
WIP [IMPAC-678] Load Widgets with Generic Templates

### PR DESCRIPTION
**Pull request by rheasunshine**
@xaun Let the fun begin

### CODE REVIEW

**Comment by cesar-tonnoir** [comment]
@xaun was it closed automatically when you deleted another branch?

**Comment by xaun** [comment]
@cesar-tonnoir yes, sorry, I have re-opened it

File: workspace/app/services/dev-settings.svc.js [hunk 1]
```diff
@@ -9,11 +9,11 @@ angular.module('impacWorkspace').service('DevSettings', function ($q, ImpacRoute
   var DEFAULTS = {
     // API Endpoints
     mnoeUrl: {
-      host: 'https://impac-mnoe-uat.maestrano.io',
+      host: 'http://localhost:7000',
```
Comment (xaun) [inline workspace/app/services/dev-settings.svc.js]: Hey @rheasunshine can you please remove this from the commit.. just squash and push -f

Comment (rheasunshine) [reply workspace/app/services/dev-settings.svc.js]: done

**Comment by cesar-tonnoir** [comment]
This has been reused and completed by @xaun in another PR - closing this one

**Pull request closed without merging**